files/class-api-client.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// phpcs:disable Universal.Files.SeparateFunctionsFromOO.Mixed
```

Comment:
Could the couple of requires and the single function be moved out of the file to maintain consistency with the expectation around classes-only being in a `class-` file?

Reply:
Definitely would be good to get around to removing the suppressions where it makes sense, but doesn't really have to be in this PR that updates the tooling to an important milestone. Can handle it separately in another PR.